tests/kernel_args/kernel_args.cpp

```diff
@@ -83,6 +104,17 @@ class TEST_NAME : public util::test_base {
           });
 
           my_queue.wait_and_throw();
+          if (error != 0) {
+            if (error & 1) {
+              FAIL(log, "kernelScalar capture error");
+            }
+            if (error & 2) {
```

Comment:
Using `(1 << 1)` and `(1 << 2)` here too for homogeneity?

Reply:
Thank you! Done in 38218e15e2d06b68bca4adccfc0ac663117384e9